src/ucs/config/global_opts.c

```diff
@@ -191,6 +191,42 @@ static ucs_config_field_t ucs_global_opts_table[] = {
   "The list order decides the priority of the providers.",
   ucs_offsetof(ucs_global_opts_t, topo_prio), UCS_CONFIG_TYPE_STRING_ARRAY},
 
+ {"DISTANCE_LAT", "phb:300ns,node:300ns,sys:500ns",
+  "Estimated latency between system devices", 0,
+  UCS_CONFIG_TYPE_KEY_VALUE(UCS_CONFIG_TYPE_TIME,
+      {"phb",
+       "connection traversing PCIe as well as a PCIe Host Bridge (typically "
+       "the CPU)",
+       ucs_offsetof(ucs_global_opts_t, dist_phb.latency)},
+      {"node",
+       "connection traversing PCIe as well as the interconnect between PCIe "
+       "Host Bridges within a NUMA node",
+       ucs_offsetof(ucs_global_opts_t, dist_node.latency)},
+      {"sys",
+       "connection traversing PCIe as well as the SMP interconnect between "
+       "NUMA nodes",
+       ucs_offsetof(ucs_global_opts_t, dist_sys.latency)},
+      {NULL}
+ )},
+
+ {"DISTANCE_BW", "phb:auto,node:17000MBs,sys:220MBs",
```

Comment:
WDYT does it worth to elaborate what "auto" means in that case?

Reply:
I think it will be redundant. I don't think that we should write all the values and formulas here. auto means that the value will be calculated automatically. so, this should be enough.